vitest.shared.config.mts

```diff
@@ -42,7 +41,11 @@ export default defineConfig({
       "/apps/demo-react-vite",
     ],
     globals: true,
-    env: loadEnv(mode, process.cwd(), ""),
+    env: {
+      ...loadEnv(mode, process.cwd(), ""),
+      FUEL_CORE_PATH: "fuels-core",
+      FORC_PATH: "fuels-forc",
+    },
```

Comment:
Is this still necessary?

Reply:
Good catch, it's not necessary anymore. I have reverted it in https://github.com/FuelLabs/fuels-ts/pull/3646/commits/66d63b5f054b738c1206a225777954564a487cb2.